test_runner/regress/test_aux_files.py

```diff
@@ -0,0 +1,81 @@
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import (
+    NeonEnv,
+    logical_replication_sync,
+)
+
+
+def test_aux_v2_config_switch(neon_simple_env: NeonEnv, vanilla_pg):
```

Comment:
Is there a particular reason why add a neon_simple_env test? Could just be a normal `neon_env_builder` test case.

Reply:
This is somehow related to `test_logical_replication`, all of them use simple_env. I could change it to the normal neon env builder.